StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -139,11 +154,31 @@ import UIKit
         }
 
         self.selection = selection
+        updateMandate()
     }
 
     func didTapAccessoryButton() {
         // TODO(porter)
     }
+
+    // MARK: Mandate handling
+    private func updateMandate(updateHeight: Bool = true) {
```

Comment:
We'll want this to animate, too, outside of the init.

Reply:
Threw this inside an animation block, I think it looks pretty good. 

https://github.com/user-attachments/assets/6d78c406-f3c3-415b-aa98-596e8bf98916